app/src/main/res/values/themes.xml

```diff
@@ -130,6 +131,10 @@
     <item name="textFieldBackground">@color/bpkWhite</item>
   </style>
 
+  <style name="LondonThemeRating">
+    <item name="ratingColor">@drawable/london_rating_colors</item>
```

Comment:
This would then be replaced by just 3 theme props

Reply:
Done